service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CloneControlledGcsBucketResourceFlight.java

```diff
@@ -104,6 +104,7 @@ public CloneControlledGcsBucketResourceFlight(
         // Destination bucket is referenced resource
         addStep(
             new SetReferencedDestinationGcsBucketInWorkingMapStep(
+                flightBeanBag.getSamService(),
```

Comment:
This looks like pure formatting change. Wonder why it showed up.

Reply:
i passes in samservice in `SetReferencedDestinationGcsBucketInWorkingMapStep` because i need to check user email from sam.